widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx

```diff
@@ -281,18 +303,19 @@ export function WalletList(props: PropTypes) {
               error={error}
             />
             <WalletNamespacesModal
-              open={!!namespacesModalState}
-              onClose={() => setNamespacesModalState(null)}
+              open={!!namespacesModalState?.isOpen}
+              onClose={handleCloseNamespaceModal}
               onConfirm={handleConfirmNamespaces}
               image={namespacesModalState?.providerImage}
               availableNamespaces={namespacesModalState?.availableNamespaces}
               singleNamespace={namespacesModalState?.singleNamespace}
             />
             <WalletDerivationPathModal
+              isOpen={!!derivationPathModalState?.isOpen}
```

Comment:
`WalletNamespacesModal` has same thing and it's `open`, here is `isOpen`. Please use same name.  (which is `open`)

Reply:
Tnx, Fixed.